docs/how-to/seed-a-db.md

```diff
@@ -0,0 +1,34 @@
+---
+id: seed-db
+title: How to Seed a database
+sidebar_label: Seed a database
+slug: /how-to/seed-db
+---
+
+# How to Seed a database
+
+You can  use a custom script to populate the database on initialization.
+
+This is done in the  [`customSeed.ts`](https://github.com/amplication/amplication/blob/master/ee/packages/amplication-git-pull-service/scripts/customSeed.ts) file. 
+
+
+```typescript
+import { PrismaClient } from '@prisma/client';
```

Comment:
I think that it is preferable to give a seeding example of a more common entity, like the one in the sample app:

```ts
const client = new PrismaClient();
  const data = {
    username: "admin",
    password: await hash("admin", bcryptSalt),
    roles: ["user"],
  };
  await client.user.upsert({
    where: { username: data.username },
    update: {},
    create: data,
  });
  void client.$disconnect();
```
AND also add a link to Prisma's docs about seeding: https://www.prisma.io/docs/guides/database/seed-database